Source: In-house
note: Recent release of the gir-scanner deprecate --strip-prefix

--- avahi-0.6.30/avahi-gobject/Makefile.am.orig	2015-04-07 16:28:55.538156105 -0700
+++ avahi-0.6.30/avahi-gobject/Makefile.am	2015-04-07 16:30:01.099587139 -0700
@@ -122,7 +122,7 @@
 Avahi_0_6_gir_CFLAGS = $(libavahi_gobject_la_CFLAGS)
 Avahi_0_6_gir_LIBS = $(lib_LTLIBRARIES)
 Avahi_0_6_gir_FILES = $(addprefix $(srcdir)/, $(CORE_SOURCES)) $(BUILT_SOURCES)
-Avahi_0_6_gir_SCANNERFLAGS = --strip-prefix=Ga
+Avahi_0_6_gir_SCANNERFLAGS = --identifier-prefix=Ga
 INTROSPECTION_GIRS += Avahi-0.6.gir
 INTROSPECTION_INSTALL_GIRS = AvahiCore-0.6.gir $(INTROSPECTION_GIRS)
 
